# Send the tracking data from Vive
string user_name
time stamp
int8 controller_channel
int8 controller_channel_sel
geometry_msgs/Pose headset
openvr_headset_ros/Vive_controller ctrl_left
openvr_headset_ros/Vive_controller ctrl_right
